machineconfiguration/v1/types.go

```diff
@@ -0,0 +1,531 @@
+package v1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	kubeletconfigv1beta1 "k8s.io/kubelet/config/v1beta1"
+)
+
+// CustomResourceDefinition for MCOConfig
```

Comment:
This should probably be dropped. other types in this repo do not have such comments.

Reply:
do you mean dropping all of the commented CRDs entirely?